settings/acceptable_versions.go

```diff
@@ -0,0 +1,51 @@
+package settings
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var acceptableVersionsCommand = &cobra.Command{
+	Use:     "acceptable-versions",
+	Short:   "Manage your pack's acceptable versions. This must be a comma seperated list of Minecraft versions, e.g. 1.16.5,1.16.4,1.16.3",
```

Comment:
The acceptable versions list is actually ordered from least to most "acceptable" - so these should probably be the other way around? It might also be a good idea to check if the versions a user provided are out of order (e.g. using FlexVer) and warn them.

Reply:
This should specify the versions are of Minecraft as well, otherwise it's a bit vague

Reply:
Ah, that also explains why in my own pack I always had it backwards lol. I'll fix that up.